src/functions-and-arrays.js

```diff
@@ -29,9 +113,29 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray (wordsArr){
+  if(wordsArr.length == 0){
+    return null;
+  }
+
+  return Array.from(new Set(wordsArr));
+}
+
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
+function doesWordExist (wordsArr, searchterm){
+  if(wordsArr.length == 0){
+    return null;
+  }
+  
+  if(wordsArr.find(word => word == searchterm) !== undefined){
```

Comment:
I THINK you can instead write:
return wordsArr.find(word => word == searchterm) !== undefined

this will either return true or false depending if the condition is met or not